From d094d3636078beca24a31b29fda269e727bb0152 Mon Sep 17 00:00:00 2001
From: Michael Cook <code@mdcook.net>
Date: Thu, 28 Dec 2017 21:41:11 -0800
Subject: [PATCH 046/249] update gems

- matched against https://pages.github.com/versions/
- Deprecation: The 'gems' configuration option has been renamed to
  'plugins'. Please update your config file accordingly.

Change-Id: I465e32d02a5877e92c5db80edd67e64d2fe66c45
---
 Gemfile      | 24 ++++++-------
 Gemfile.lock | 95 +++++++++++++++++++++++++++-------------------------
 _config.yml  |  2 +-
 3 files changed, 62 insertions(+), 59 deletions(-)

diff --git a/Gemfile b/Gemfile
index 4813874..706d7ee 100644
--- a/Gemfile
+++ b/Gemfile
@@ -1,31 +1,31 @@
 source "https://rubygems.org"
 
 
-gem "jekyll", "3.4.5"
+gem "jekyll", "3.6.2"
 gem "therubyracer", "0.12.3"
 
 
 group :jekyll_plugins do
-   gem 'activesupport', '= 4.2.8'
+   gem 'activesupport', '= 4.2.9'
    gem 'github-pages-health-check', '= 1.3.5'
-   gem 'jekyll-avatar', '= 0.4.2'
-   gem 'jekyll-coffeescript', '= 1.0.1'
+   gem 'jekyll-avatar', '= 0.5.0'
+   gem 'jekyll-coffeescript', '= 1.0.2'
    gem 'jekyll-feed', '= 0.9.2'
-   gem 'jekyll-gist', '= 1.4.0'
+   gem 'jekyll-gist', '= 1.4.1'
    gem 'jekyll-mentions', '= 1.2.0'
    gem 'jekyll-paginate', '= 1.1.0'
    gem 'jekyll-redirect-from', '= 0.12.1'
-   gem 'jekyll-relative-links', '= 0.4.1'
+   gem 'jekyll-relative-links', '= 0.5.2'
    gem 'jekyll-sass-converter', '= 1.5.0'
-   gem 'jekyll-seo-tag', '= 2.2.3'
-   gem 'jekyll-sitemap', '= 1.0.0'
+   gem 'jekyll-seo-tag', '= 2.3.0'
+   gem 'jekyll-sitemap', '= 1.1.1'
    gem 'jekyll-swiss', '= 0.4.0'
-   gem 'jemoji', '= 0.8.0'
-   gem 'kramdown', '= 1.13.2'
-   gem 'liquid', '= 3.0.6'
+   gem 'jemoji', '= 0.8.1'
+   gem 'kramdown', '= 1.14.0'
+   gem 'liquid', '= 4.0.0'
    gem 'listen', '= 3.0.6'
    gem 'mercenary', '~> 0.3'
    gem 'minima', '= 2.1.1'
-   gem 'rouge', '= 1.11.1'
+   gem 'rouge', '= 2.2.1'
    gem 'terminal-table', '~> 1.4'
 end
diff --git a/Gemfile.lock b/Gemfile.lock
index 73f4b3f..91ba7ad 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -1,22 +1,23 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.8)
+    activesupport (4.2.9)
       i18n (~> 0.7)
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.5.1)
-      public_suffix (~> 2.0, >= 2.0.2)
+    addressable (2.5.2)
+      public_suffix (>= 2.0.2, < 4.0)
     coffee-script (2.4.1)
       coffee-script-source
       execjs
-    coffee-script-source (1.12.2)
+    coffee-script-source (1.11.1)
     colorator (1.1.0)
-    ethon (0.10.1)
+    concurrent-ruby (1.0.5)
+    ethon (0.11.0)
       ffi (>= 1.3.0)
     execjs (2.7.0)
-    faraday (0.12.2)
+    faraday (0.13.1)
       multipart-post (>= 1.2, < 3)
     ffi (1.9.18)
     forwardable-extended (2.6.0)
@@ -27,28 +28,30 @@ GEM
       octokit (~> 4.0)
       public_suffix (~> 2.0)
       typhoeus (~> 0.7)
-    html-pipeline (2.6.0)
+    html-pipeline (2.7.1)
       activesupport (>= 2)
       nokogiri (>= 1.4)
-    i18n (0.8.6)
-    jekyll (3.4.5)
+    i18n (0.9.1)
+      concurrent-ruby (~> 1.0)
+    jekyll (3.6.2)
       addressable (~> 2.4)
       colorator (~> 1.0)
       jekyll-sass-converter (~> 1.0)
       jekyll-watch (~> 1.1)
-      kramdown (~> 1.3)
-      liquid (~> 3.0)
+      kramdown (~> 1.14)
+      liquid (~> 4.0)
       mercenary (~> 0.3.3)
       pathutil (~> 0.9)
-      rouge (~> 1.7)
+      rouge (>= 1.7, < 3)
       safe_yaml (~> 1.0)
-    jekyll-avatar (0.4.2)
+    jekyll-avatar (0.5.0)
       jekyll (~> 3.0)
-    jekyll-coffeescript (1.0.1)
+    jekyll-coffeescript (1.0.2)
       coffee-script (~> 2.2)
+      coffee-script-source (~> 1.11.1)
     jekyll-feed (0.9.2)
       jekyll (~> 3.3)
-    jekyll-gist (1.4.0)
+    jekyll-gist (1.4.1)
       octokit (~> 4.2)
     jekyll-mentions (1.2.0)
       activesupport (~> 4.0)
@@ -57,49 +60,49 @@ GEM
     jekyll-paginate (1.1.0)
     jekyll-redirect-from (0.12.1)
       jekyll (~> 3.3)
-    jekyll-relative-links (0.4.1)
+    jekyll-relative-links (0.5.2)
       jekyll (~> 3.3)
     jekyll-sass-converter (1.5.0)
       sass (~> 3.4)
-    jekyll-seo-tag (2.2.3)
+    jekyll-seo-tag (2.3.0)
       jekyll (~> 3.3)
-    jekyll-sitemap (1.0.0)
+    jekyll-sitemap (1.1.1)
       jekyll (~> 3.3)
     jekyll-swiss (0.4.0)
-    jekyll-watch (1.5.0)
-      listen (~> 3.0, < 3.1)
-    jemoji (0.8.0)
-      activesupport (~> 4.0)
+    jekyll-watch (1.5.1)
+      listen (~> 3.0)
+    jemoji (0.8.1)
+      activesupport (~> 4.0, >= 4.2.9)
       gemoji (~> 3.0)
       html-pipeline (~> 2.2)
       jekyll (>= 3.0)
-    kramdown (1.13.2)
-    libv8 (3.16.14.19-x86_64-linux)
-    liquid (3.0.6)
+    kramdown (1.14.0)
+    libv8 (3.16.14.19)
+    liquid (4.0.0)
     listen (3.0.6)
       rb-fsevent (>= 0.9.3)
       rb-inotify (>= 0.9.7)
     mercenary (0.3.6)
-    mini_portile2 (2.2.0)
+    mini_portile2 (2.3.0)
     minima (2.1.1)
       jekyll (~> 3.3)
     minitest (5.10.3)
     multipart-post (2.0.0)
     net-dns (0.8.0)
-    nokogiri (1.8.0)
-      mini_portile2 (~> 2.2.0)
-    octokit (4.7.0)
+    nokogiri (1.8.1)
+      mini_portile2 (~> 2.3.0)
+    octokit (4.8.0)
       sawyer (~> 0.8.0, >= 0.5.3)
-    pathutil (0.14.0)
+    pathutil (0.16.1)
       forwardable-extended (~> 2.6)
     public_suffix (2.0.5)
     rb-fsevent (0.10.2)
     rb-inotify (0.9.10)
       ffi (>= 0.5.0, < 2)
     ref (2.0.0)
-    rouge (1.11.1)
+    rouge (2.2.1)
     safe_yaml (1.0.4)
-    sass (3.5.1)
+    sass (3.5.4)
       sass-listen (~> 4.0.0)
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
@@ -115,7 +118,7 @@ GEM
     thread_safe (0.3.6)
     typhoeus (0.8.0)
       ethon (>= 0.8.0)
-    tzinfo (1.2.3)
+    tzinfo (1.2.4)
       thread_safe (~> 0.1)
     unicode-display_width (1.3.0)
 
@@ -123,30 +126,30 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
-  activesupport (= 4.2.8)
+  activesupport (= 4.2.9)
   github-pages-health-check (= 1.3.5)
-  jekyll (= 3.4.5)
-  jekyll-avatar (= 0.4.2)
-  jekyll-coffeescript (= 1.0.1)
+  jekyll (= 3.6.2)
+  jekyll-avatar (= 0.5.0)
+  jekyll-coffeescript (= 1.0.2)
   jekyll-feed (= 0.9.2)
-  jekyll-gist (= 1.4.0)
+  jekyll-gist (= 1.4.1)
   jekyll-mentions (= 1.2.0)
   jekyll-paginate (= 1.1.0)
   jekyll-redirect-from (= 0.12.1)
-  jekyll-relative-links (= 0.4.1)
+  jekyll-relative-links (= 0.5.2)
   jekyll-sass-converter (= 1.5.0)
-  jekyll-seo-tag (= 2.2.3)
-  jekyll-sitemap (= 1.0.0)
+  jekyll-seo-tag (= 2.3.0)
+  jekyll-sitemap (= 1.1.1)
   jekyll-swiss (= 0.4.0)
-  jemoji (= 0.8.0)
-  kramdown (= 1.13.2)
-  liquid (= 3.0.6)
+  jemoji (= 0.8.1)
+  kramdown (= 1.14.0)
+  liquid (= 4.0.0)
   listen (= 3.0.6)
   mercenary (~> 0.3)
   minima (= 2.1.1)
-  rouge (= 1.11.1)
+  rouge (= 2.2.1)
   terminal-table (~> 1.4)
   therubyracer (= 0.12.3)
 
 BUNDLED WITH
-   1.15.1
+   1.16.1
diff --git a/_config.yml b/_config.yml
index 465fbc8..9f93dcb 100644
--- a/_config.yml
+++ b/_config.yml
@@ -101,5 +101,5 @@ description: "The LineageOS Wiki"
 # needed for sitemap.xml file only
 url: https://wiki.lineageos.org
 
-gems:
+plugins:
   - jekyll-redirect-from
\ No newline at end of file
-- 
2.17.1

